duckduckgo/iOS
# Fix omnibar width on tablet

Reviewer: Chris
Asana: https://app.asana.com/0/414709148257752/422862278695032

**Description**:
Fix omnibar size on split screen tablet

**Steps to test this PR**:
1. Launch app on tablet
2. Launch a split screen application
3. Omnibar width should be correct


###### Reviewer Checklist:


###### PR DRI Checklist:
